Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/Style/CustomTextFieldStyle.swift

```diff
@@ -0,0 +1,28 @@
+//
+
+import SwiftUI
+
+struct CustomTextFieldStyle: TextFieldStyle {
+    private let isEnabled: Bool
+    
+    init(isEnabled: Bool = true) {
+        self.isEnabled = isEnabled
+    }
+    
+    func _body(configuration: TextField<Self._Label>) -> some View {
```

Comment:
Don't know much about SwiftUI, but why is it `_body` here and `makeBody` in `CustomButtonStyle`? Just curious.

Reply:
> Don't know much about SwiftUI, but why is it `_body` here and `makeBody` in `CustomButtonStyle`? Just curious.

To be honest, me neither :) but here is a bit of explanation of underscored properties in SwiftUI https://stackoverflow.com/a/65209375 don't know yet too much about SwiftUI but I guess this is a good point in time to learn more about it :D

Reply:
Swift nowadays is a hell of a learning ride! 🤪

Reply:
true story bro! :D